src/app/application.cpp

```diff
@@ -771,6 +795,64 @@ void Application::shutdownCleanup(QSessionManager &manager)
 }
 #endif
 
+#ifdef Q_OS_WIN
+namespace
+{
+    std::uint64_t autoMemoryWorkingSetSize(const std::uint64_t memorySize)
```

Comment:
Those should be `int64_t`.

Reply:
> Those should be `int64_t`.

Those are assigned by/to unsigned types.

Reply:
(I don't insist) I will only convert to unsigned type at the very last moment.

Reply:
> (I don't insist) I will only convert to unsigned type at the very last moment.

So you would first conver input data coming from WinAPI from unsigned and pass to this function and then convert result to unsigned again?